R/create_covariate_summary_plot.R

```diff
@@ -0,0 +1,158 @@
+#' Create the covariate summary plot 
+#' https://rdrr.io/github/audrey-b/BUGSnet/man/data.plot.html
+#'
+#' @param all_data Study data including covariate columns, in wide or long format
+#' @param metaoutcome Reactive containing meta analysis outcome: "Continuous" or "Binary"
+#' @param covariate_or_baseline Output from widget to toggle between covariate and baseline risk for the plot type.
+#' @param treatment_df Reactive containing data frame containing treatment IDs (Number) and names (Label)
```

Comment:
These parameters (metaoutcome & treatment_df) aren't reactives, they're the data itself.